transcom/mymove
# INT B-20011-13-14 (Follow Up)

## [B-20011](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20011) & [B-20013](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20013) & [B-20014](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20014)

## Summary

Updates the HQ page to use the columns functions from the various existing queues rather than it's own copy. Having a second copy to keep up to date was causing issues since the HQ role hasn't been merged into main.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as an HQ user
3. Tab through all the queue and verify they still render.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
